pkgs/top-level/all-packages.nix

```diff
@@ -17857,7 +17857,7 @@ in
   kdevelop-pg-qt = libsForQt5.callPackage ../applications/editors/kdevelop5/kdevelop-pg-qt.nix {};
 
   kdevelop = libsForQt5.callPackage ../applications/editors/kdevelop5/kdevelop.nix {
-    llvmPackages = llvmPackages_38;
+    llvmPackages = llvmPackages_7;
```

Comment:
Any particular reason not to use `llvmPackages_latest`? Is this likely to break with updates?

Reply:
The llvm package provides the c++ parsing and analysis for kdevelop and different versions of llvm can dramatically impact how the application behaves. The decision to move from one version of llvm to another might not break compilation, but it could break functionality, so a person should specifically test any llvm bumps to avoid breaking behavior in the application.

Reply:
Alright. Do you think this PR needs more testing? I don't use kdevelop personally.